src/pd/client.rs

```diff
@@ -153,8 +152,20 @@ impl RpcClient {
             core: Mutex::new(RpcClientCore::new(endpoints)),
             cluster_id: 0,
         };
-        client.cluster_id = try!(client.get_cluster_id());
-        Ok(client)
+
+        for _ in 0..MAX_PD_SEND_RETRY_COUNT {
```

Comment:
I think 1s is fine, we only retry here when TiKV starts and cannot connect to PD. PD may not ready to serve yet, no need to retry in a very short time.